packages/db/src/runtime/db-client.ts

```diff
@@ -36,6 +38,42 @@ const remoteResultSchema = z.object({
 });
 
 export function createRemoteDatabaseClient(appToken: string, remoteDbURL: string) {
+	const remoteUrl = new URL(remoteDbURL);
+
+	switch (remoteUrl.protocol) {
+		case 'http:':
+		case 'https:':
+			return createStudioDatabaseClient(appToken, remoteUrl);
+		case 'libsql+http:':
+			remoteUrl.protocol = 'http:';
+		case 'libsql+https:':
```

Comment:
The fallthrough here will mean the `http:` case will always be overwritten with `https:`. This is a great example of how easy it is for bugs to sneak in when using switch fallthrough like this. I think it's best if you can refactor this.